CHANGELOG.md

```diff
@@ -74,6 +74,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `OutOfStock` style and success message. (#399)
 - Apply new local tokens to `Button` (#442)
 - Gather all `Button` variants in the folder (`ButtonBuy`, `ButtonLink`, `ButtonIcon`, `ButtonSignIn`) (#442)
+- Apply new local tokens to `Badge` (#462)
```

Comment:
could you move this to the `Unreleased` section on top of the `Changed` list, as it wasn't released yet?